src/components/HowTo/HowTo.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Box, Container, Stack, Typography } from '@mui/material';
+import howToBackgroundDark from '../../assets/images/how-to-background-dark.jpeg';
+import howToBackgroundLight from '../../assets/images/how-to-background-light.jpeg';
+import { HowToStep } from './HowToStep';
+
+export interface Step {
+  Icon?: React.ReactNode;
+  title: string;
+  description: string | React.ReactNode;
+  link?: {
+    href: string;
+    label: string;
+    target?: React.HTMLAttributeAnchorTarget;
+  };
+}
+
+export interface HowToProps {
+  title: string;
+  steps: Step[];
+  theme: 'light' | 'dark';
+
+  rowMaxSteps?: number;
+  stepsAlignment?: 'right' | 'center' | 'left';
+}
+
+const groupStepsByRows = (steps: Step[], rowMaxSteps: number): Step[][] => {
+  const numSteps = steps.length;
+  return new Array(Math.ceil(numSteps / rowMaxSteps))
+    .fill(undefined)
+    .map((_, i) => steps.slice(i * rowMaxSteps, i * rowMaxSteps + rowMaxSteps));
+};
+
+export const HowTo: React.FC<HowToProps> = ({
+  title,
+  steps,
+  theme,
+  rowMaxSteps = 4,
+  stepsAlignment = 'center',
+}) => {
+  const isDarkTheme = theme === 'dark';
+  const background = isDarkTheme ? howToBackgroundDark : howToBackgroundLight;
+
+  const alignment = { center: 'center', left: 'start', right: 'end' }[
+    stepsAlignment
+  ];
+
+  console.time();
```

Comment:
Is it necessary this console.time & console.timeEnd?

Reply:
Ups, forgot to remove it